website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -66,9 +66,8 @@ Once you install the dbt Cloud CLI, you need to configure it to connect to a dbt
     ```yaml
     # dbt_project.yml
     name:
-
     version:
-    ...
+    # Your project configs...
```

Comment:
adding this so it's clear to the user that their project info goes here cc @cesher and thank you @runleonarun